src/components/BB-BUS/TextSearch.js

```diff
@@ -3,10 +3,12 @@ import React, { memo } from 'react';
 import { StyleSheet, View } from 'react-native';
 import { SearchBar } from 'react-native-elements';
 
-import { colors, normalize } from '../../config';
+import { colors, consts, normalize, texts } from '../../config';
 import { Label } from '../Label';
 import { WrapperHorizontal } from '../Wrapper';
 
+const a11yText = consts.a11yLabel;
```

Comment:
why not `const { a11yLabel } = consts;`?

Reply:
It is used in two different ways in the code. We will need to refactor the whole code.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/59c9c808be9e127b5416245dcda7109ca46cf86c

Reply:
correct, but than let's use the favorite one if we add new code